### PR TITLE
193-add-soft-delete-to-cars-and-deliveries

### PR DESCRIPTION
**Pull request by NastyaBondarenko**
1. Add soft delete to cars and deliveries
2. Refactoring tests according to it

### CODE REVIEW

File: goods-partner-backend/src/main/java/com/goodspartner/entity/Delivery.java [hunk 1]
```diff
@@ -40,6 +57,8 @@ public class Delivery {
     @Column(length = 9)
     private DeliveryStatus status;
 
+    private boolean deleted = Boolean.FALSE;
```
Comment (BogdanParashchak) [inline goods-partner-backend/src/main/java/com/goodspartner/entity/Delivery.java:60]: can we use primitive instead?

Comment (BogdanParashchak) [inline goods-partner-backend/src/main/java/com/goodspartner/entity/Delivery.java:60]: make the same style like for other variables (mark with @Column)

**Review by airavata-elephant — APPROVED** [review]

File: goods-partner-backend/src/main/java/com/goodspartner/entity/Delivery.java (hunk 1 above)
Comment (NastyaBondarenko) [reply goods-partner-backend/src/main/java/com/goodspartner/entity/Delivery.java]: +

Comment (NastyaBondarenko) [reply goods-partner-backend/src/main/java/com/goodspartner/entity/Delivery.java]: yes, both variants works, replaced to primitive

**Review by Eugene7011 — APPROVED** [review]

**Pull request merged**